auth-web/src/views/auth/BusinessProfileView.vue

```diff
@@ -113,7 +113,8 @@ export default class BusinessProfileView extends Mixins(AccountChangeMixin, Next
 
   async mounted () {
     this.isLoading = true
-    // Include businessid from query string in case session is empty.
+    // Pass in business id from URL query string.
+    // Note: action will set session storage.
     await this.loadBusiness(this.$attrs.businessid)
```

Comment:
This is also called from here, so I copied the comment over.

https://github.com/bcgov/sbc-auth/blob/9005657e59620d4c8076dfe090f5774550222604/auth-web/src/views/auth/staff/StaffDashboardView.vue#L469